### PR TITLE
Added Timespan component

### PR DESCRIPTION
**Pull request by fernandodonaire**
Screen Shot 2020-11-12 at 1 17 11 PM

Can be tested by: 

1) Pulling down this branch. 
2) Running `npm install`.
3) Running the app locally.
4) On the Home page, clicking on the different buttons on the timespan.

### CODE REVIEW

File: src/components/Timespan/index.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import React from "react";
```
Comment (SpencerMarks-IDHA) [inline src/components/Timespan/index.js:1]: weird javascript syntax but whatever

File: src/components/Timespan/index.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+import { bindActionCreators } from "redux";
+import { changeDateRange } from "../../actions/filters";
+import { getNumberOfWeeksBetweenDates } from "../../utils/dateHelpers";
+import { TimespanButtonsWrapper } from "../styled-components/TimespanButtonsWrapper";
+import { Button, ButtonLink } from "../styled-components/Button";
+import { ebolaInitialDateRange } from "../../constants/DateRanges";
+
+const Timespan = (props) => {
+  const updateDateRange = (number = 0, unitOfTime) => {
```
Comment (SpencerMarks-IDHA) [inline src/components/Timespan/index.js:13]: would documenting these methods be a good idea or is it not the convention in JS to document functions?

File: src/components/Timespan/index.js (hunk 1 above)
Comment (fernandodonaire) [reply src/components/Timespan/index.js:1]: Yeah that's just how you import the React library in JS ES6: https://create-react-app.dev/docs/importing-a-component/

File: src/components/Timespan/index.js (hunk 2 above)
Comment (fernandodonaire) [reply src/components/Timespan/index.js:13]: What I typically do is add comments inside of the function block if something is not clear. In this case I added comments on lines 14,18 and 20 inside of the `updateDateRange()` function to say what each line is doing.

**Review by SpencerMarks-IDHA — APPROVED** [review]
please update that comment but other looks great.

File: src/components/Timespan/index.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    // Uses the dayjs library to add units of time to the 'from' date in the initial date range.
+    const newToDate = new Date(
+      dayjs(ebolaInitialDateRange.from).add(number, unitOfTime).format()
+    );
+    // Updates the date range filter in the redux state.
+    props.changeDateRange([ebolaInitialDateRange.from, newToDate]);
+    // Updates the sliderRange in the DateRangeSlider component.
```
[the remark this answers is not in the document]
Comment (fernandodonaire) [reply src/components/Timespan/index.js:20]: Thanks @SpencerMarks-IDHA ! Good catch. I'll update that comment.

**Pull request merged**